maintainers/metrics-maintainer.nix

```diff
@@ -0,0 +1,101 @@
+/*
+  Count how many derivations are maintained by the Nix@NGI team in Nixpkgs.
+
+  Usage:
+
+  # NGIpkgs commit
+  nix eval -f ./metrics-maintainer.nix metrics.maintained --argstr team ngi --argstr rev 4d6c17428e6e32f474e8fc75a87b8f9bd6c6356d
+
+  # release tag
+  nix eval -f ./metrics-maintainer.nix metrics.maintained --argstr team ngi --argstr rev 25.02
+
+  # branch
+  nix eval -f ./metrics-maintainer.nix metrics.maintained --argstr team ngi --argstr rev main
+
+  # latest Nixpkgs
+  nix eval -f ./metrics-maintainer.nix metrics.maintained --argstr team ngi --arg nixpkgs 'fetchTarball "https://github.com/NixOS/nixpkgs/tarball/master"'
+*/
+{
+  rev ? "main",
+  ngipkgs ? import (fetchTarball "https://github.com/ngi-nix/ngipkgs/tarball/${rev}") { },
```

Comment:
wouldn't it make more sense for this to default to the repo path but then we can override it manually to a rev/tag if we need to instead of it being the default?